.github/workflows/r.yml

```diff
@@ -170,33 +170,28 @@ jobs:
         continue-on-error: true
         run: archery docker push r
 
-  windows:
-    name: AMD64 Windows RTools ${{ matrix.rtools }}
+  windows-cpp:
+    name: AMD64 Windows C++ RTools ${{ matrix.config.rtools }} ${{ matrix.config.arch }}
     runs-on: windows-latest
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
     timeout-minutes: 60
     strategy:
       fail-fast: false
       matrix:
-        rtools: [35, 40]
-    env:
-      ARROW_R_CXXFLAGS: "-Werror"
-      _R_CHECK_TESTS_NLINES_: 0
+        config:
+        # We use the R version that was released at the time of the arrow release in order
+        # to make sure we can download binaries from RSPM.
```

Comment:
I think this comment is left over from the backwards compatibility matrix?

Reply:
Oops yes, I was copy pasting from a few sources and that was one